src/scripts.js

```diff
@@ -37,19 +37,19 @@ const loginView = document.getElementById('loginView');
 const errorMessageContainer = document.getElementById('errorMessageContainer');
 const submitBookingButtonSection = document.getElementById('submitBookingButtonSection');
 
-// window.onload = validateLogin();
+window.onload = validateLogin();
 addNewBookingsButton.addEventListener('click', renderNewBookingsView);
 searchCalendar.addEventListener('click', showAvailableRooms);
 filterRoomTypeButton.addEventListener('click', showFilteredRooms);
 goBackButton.addEventListener('click', determineViewToGoBackTo);
 availableRoomsSection.addEventListener('click', displayClickedRoom);
 // not sure how to deal with this yet....
-availableRoomsSection.addEventListener('keyup', displayClickedRoom);
+availableRoomsSection.addEventListener('keydown', displayClickedRoom);
```

Comment:
Curious to know if this worked out as the way you expected, if it is working as expected, remove the comment above.